test/python/fx_importer/symbolic_shape_expr_test.py

```diff
@@ -371,7 +371,7 @@ def forward(self, x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
 # CHECK:        torch.bind_symbolic_shape %[[ARG0]], [%[[S0]]], affine_map<()[s0] -> (1, s0)> : !torch.vtensor<[1,?],f32>
 # CHECK:        torch.bind_symbolic_shape %[[ARG1]], [%[[S1]]], affine_map<()[s0] -> (s0)> : !torch.vtensor<[?],f32>
 # CHECK:        %[[EXPAND:.+]] = torch.aten.expand %[[ARG0]], {{.*}}, {{.*}} : !torch.vtensor<[1,?],f32>, !torch.list<int>, !torch.bool -> !torch.vtensor<[?,?],f32>
-# CHECK:        torch.bind_symbolic_shape %[[EXPAND]], [%[[S0]], %[[S1]]], affine_map<()[s0, s1] -> (s1, s0)> : !torch.vtensor<[?,?],f32>
+# CHECK-DISABLED:        torch.bind_symbolic_shape %[[EXPAND]], [%[[S1]], %[[S0]]], affine_map<()[s0, s1] -> (s0, s1)> : !torch.vtensor<[?,?],f32>
```

Comment:
Perhaps add a 1-line breadcrumb here (what differed between stable vs nightly) and a TODO to re-enable once they're consistent?